docs/schema.md

```diff
@@ -727,6 +728,7 @@ The known reference `type` values are:
 	"credits": [ {
 		"name": string,
 		"contact": [ string ],
+    "type": [ string ],
```

Comment:
I have no idea why GH is displaying this line like that.  I even copy/pasted the indentation from previous lines to try to match it 😭